test/test_cost.py

```diff
@@ -481,7 +481,10 @@ def _create_seq_mock_data_dqn(
     @pytest.mark.parametrize("device", get_default_devices())
     @pytest.mark.parametrize("action_spec_type", ("one_hot", "categorical"))
     @pytest.mark.parametrize("td_est", list(ValueEstimators) + [None])
-    def test_dqn(self, delay_value, double_dqn, device, action_spec_type, td_est):
+    @pytest.mark.parametrize("reduction", [None, "none", "mean", "sum"])
```

Comment:
I think at this stage we must be careful not to overparametrize the tests. Maybe we could just run a dedicated test for DQN that runs it on a simple unique scenario and checks that the loss values all have the right shape?